pipelines/templates/buildpack.yml.erb

```diff
@@ -101,7 +101,7 @@ resources: #####################################################################
   - name: cf-deployment
     type: git
     source:
-      uri: https://github.com/cloudfoundry/cf-deployment
```

Comment:
typo

Reply:
wow that's a bonafide typo!

Reply:
clearly i was trying to run the `gh` command in my terminal 😅